main.py

```diff
@@ -1,10 +1,47 @@
-import init_django_orm  # noqa: F401
+import json
 
+import init_django_orm  # noqa: F401
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    try:
+        with open("players.json", "r") as file:
+            players_data = json.load(file)
+    except FileNotFoundError:
+        print("players.json file not found.")
```

Comment:
After printing the error message for FileNotFoundError, consider adding a return statement to exit the main function early. This will prevent the code from attempting to iterate over 'players_data' when the file is not found, which would raise an UnboundLocalError.